### ISSUE TITLE
Override bundled DXVK

### ISSUE THREAD

**Issue by emansom** [Open-Wine-Components/ULWGL#165]
Is there any method to override the bundled DXVK for testing etc?

**Comment by R1kaB3rN** [comment]
One method to override the bundled DXVK is to modify the wine prefix, which comes with the advantage of maintaining the integrity of the Proton build. Since https://github.com/Open-Wine-Components/umu-launcher/pull/109, winetricks support has been implemented, requiring either GE-Proton or UMU-Proton to use. See the man page for an example.

Also, please note that the DXVK version reported from `DXVK_HUD=version` should be reported inaccurately as I believe DXVK uses its read-only `version` file to report its version.

**Comment by emansom** [comment]
When installing DXVK to the regular Wine prefix, it gets overriden by the DXVK bundled with UMU-Proton.

```
./.local/share/Steam/compatibilitytools.d/UMU-Proton-9.0-2/files/lib64/wine/x86_64-windows/d3d11.dll
./.local/share/Steam/compatibilitytools.d/UMU-Proton-9.0-2/files/lib64/wine/dxvk/d3d11.dll
./.local/share/Steam/compatibilitytools.d/UMU-Proton-9.0-2/files/share/default_pfx/drive_c/windows/syswow64/d3d11.dll
./.local/share/Steam/compatibilitytools.d/UMU-Proton-9.0-2/files/share/default_pfx/drive_c/windows/system32/d3d11.dll
./.local/share/Steam/compatibilitytools.d/UMU-Proton-9.0-2/files/lib/wine/i386-windows/d3d11.dll
./.local/share/Steam/compatibilitytools.d/UMU-Proton-9.0-2/files/lib/wine/dxvk/d3d11.dll
```

The paths they are located within have read only bits as well, so someone doesn't like me replacing them I figure.

For context, i'm using the latest stable Lutris (arch repo, `0.5.17-5`) with UMU-Proton and have set the version of DXVK to "Manual" in Lutris' Runner options and this doesn't do much.

**Comment by R1kaB3rN** [comment]
> When installing DXVK to the regular Wine prefix, it gets overriden by the DXVK bundled with UMU-Proton.
> 

Indeed. If the verb for Proton isn't `runinprefix`, the relevant DLLs will be copied again. To avoid this, `PROTON_VERB=runinprefix` must be set after applying the DXVK verb.

However, even if that problem is solved, I suppose this isn't your use case -- I'm guessing you want to use a bleeding edge or your own customized DXVK build?

**Comment by emansom** [comment]
> I'm guessing you want to use a bleeding edge or your own customized DXVK build?

Yes.

**Comment by LovingMelody** [comment]
Would be interested in seeing this expanded to dxvk-nvapi as well, not just dxvk
